app/src/main/java/ar/com/wolox/android/example/model/User.kt

```diff
@@ -0,0 +1,3 @@
+package ar.com.wolox.android.example.model
+
+data class User(val uid: String, val email: String, val name: String, val nickname: String)
```

Comment:
Same here.
Also, check the login's response, you'll find some more fields to store.